storage/backend/ceph/ceph.go

```diff
@@ -150,7 +150,7 @@ func (c *Driver) Format(do storage.DriverOptions) error {
 
 // Destroy a volume.
 func (c *Driver) Destroy(do storage.DriverOptions) error {
-	poolName := do.Volume.Params["pool"]
+	poolName := do.Volume.Params["pool"].(string)
```

Comment:
if you do this and `do.Volume.Params["pool"]` is *not* a string, this will crash the program.

Please take it like so:

```go
poolName, ok := do.Volume.Params["pool"].(string)
if !ok {
  // generate an error
}
```

This will allow you to trap any conversion error.

Again, I can't express how important it is to not perform type assertions this way.

Please correct it in all the code.